new-integration-tests/src/test/resources/exporter/exporter-config.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
rename the file name to exporter-config.yaml to be consistent with yaml file extension with other similar files in resource directory.

Reply:
done